sidebars.js

```diff
@@ -13,11 +13,6 @@ module.exports = {
           "developer/java/patterns/java_swagger_annotations",
         ],
       },
-      {
-        type: "category",
-        label: "Azure Development Patterns",
-        items: [],

```

Comment:
Docusaurus does not allow empty items - this does not show in the docs, even on live